models/category/src/category-draft/presets/sample-data-goodstore/room-decor.ts

```diff
@@ -0,0 +1,31 @@
+import {
+  LocalizedString,
+  KeyReference,
+} from '@commercetools-test-data/commons';
+import type { TCategoryDraftBuilder, TCategoryDraft } from '../../../types';
+import empty from '../empty';
+import homeDecor from './home-decor';
+
+const homeDecorDraft = homeDecor().build<TCategoryDraft>();
+
+const roomDecor = (): TCategoryDraftBuilder =>
+  empty()
+    .name(
+      LocalizedString.presets
+        .empty()
+        ['en-GB']('Room Decor')
+        ['en-US']('Room Decor')
+        ['de-DE']('Zimmerdekoration')
+    )
+    .slug(
+      LocalizedString.presets
+        .empty()
+        ['en-GB']('room-decor')
+        ['en-US']('room-decor')
+        ['de-DE']('zimmerdekoration')
```

Comment:
There was no slug in the source data, is the pattern to copy the translation or make them all identical? Just making sure